### ISSUE TITLE
Commented attribute fails the generator

### ISSUE THREAD

**Issue by enikao** [ModelingValueGroup/DclareForMPS#42]
I commented a rule attribute with the default commenting feature:

````
/*attr set<struct<Triple>> triples := []; 
*/
````

The generator doesn't like that:
````
jetbrains.mps.lang.smodel.generator.smodelAdapter.NodeCastException: Can't cast node: 3160209179995890717, concept: jetbrains.mps.lang.core.structure.BaseCommentAttribute to concept: DclareMPS.structure.RuleSet
	at jetbrains.mps.lang.smodel.generator.smodelAdapter.SNodeOperations.cast(SNodeOperations.java:563)
	at DclareMPS.behavior.Attribute__BehaviorDescriptor.getContextType_id29R9$zZWsRH(Attribute__BehaviorDescriptor.java:67)
	at DclareMPS.behavior.Attribute__BehaviorDescriptor.invokeSpecial0(Attribute__BehaviorDescriptor.java:106)
	at jetbrains.mps.core.aspects.behaviour.BaseBHDescriptor.invokeSpecial(BaseBHDescriptor.java:358)
	at jetbrains.mps.core.aspects.behaviour.BaseBHDescriptor.invokeVirtual(BaseBHDescriptor.java:317)
	at jetbrains.mps.core.aspects.behaviour.BaseBHDescriptor.invoke(BaseBHDescriptor.java:241)
	at jetbrains.mps.core.aspects.behaviour.SMethodImpl.invoke0(SMethodImpl.java:151)
	at jetbrains.mps.core.aspects.behaviour.SMethodImpl.invoke(SMethodImpl.java:126)
	at main.QueriesGenerated.sourceNodeQuery_1_10(QueriesGenerated.java:516)
	at main.QueriesGenerated$SNQ.evaluate(QueriesGenerated.java:1269)
	at jetbrains.mps.generator.template.DefaultQueryExecutionContext.evaluate(DefaultQueryExecutionContext.java:136)
	at jetbrains.mps.generator.impl.TemplateProcessor$MacroWithInput.getNewInputNode(TemplateProcessor.java:291)
	at jetbrains.mps.generator.impl.TemplateProcessor$CopySrcMacros.apply(TemplateProcessor.java:401)
	at jetbrains.mps.generator.impl.TemplateProcessor.applyMacro(TemplateProcessor.java:112)
	at jetbrains.mps.generator.impl.TemplateProcessor.applyTemplate(TemplateProcessor.java:137)
	at jetbrains.mps.generator.impl.TemplateProcessor$MacroImpl.nextMacro(TemplateProcessor.java:263)
	at jetbrains.mps.generator.impl.TemplateProcessor$VarMacro2.apply(TemplateProcessor.java:611)
	at jetbrains.mps.generator.impl.TemplateProcessor.applyMacro(TemplateProcessor.java:112)
	at jetbrains.mps.generator.impl.TemplateProcessor.apply(TemplateProcessor.java:102)
	at jetbrains.mps.generator.impl.TemplateContainer.apply(TemplateContainer.java:77)
	at jetbrains.mps.generator.impl.RuleConsequenceProcessor$TemplateConsequence.processRuleConsequence(RuleConsequenceProcessor.java:249)
	at jetbrains.mps.generator.impl.interpreted.TemplateReductionRuleInterpreted.apply(TemplateReductionRuleInterpreted.java:74)
	at jetbrains.mps.generator.template.DefaultQueryExecutionContext.applyRule(DefaultQueryExecutionContext.java:219)
	at jetbrains.mps.generator.impl.TemplateExecutionEnvironmentImpl.tryToReduce(TemplateExecutionEnvironmentImpl.java:412)
	at jetbrains.mps.generator.impl.TemplateExecutionEnvironmentImpl.tryToReduce(TemplateExecutionEnvironmentImpl.java:349)
	at jetbrains.mps.generator.impl.TemplateGenerator$FullCopyFacility.copyInputNode(TemplateGenerator.java:1208)
	at jetbrains.mps.generator.impl.TemplateGenerator$FullCopyFacility.copyInputNode(TemplateGenerator.java:1226)
	at jetbrains.mps.generator.impl.TemplateGenerator$FullCopyFacility.copyInputNode(TemplateGenerator.java:1226)
	at jetbrains.mps.generator.impl.TemplateGenerator$FullCopyFacility.copyInputNode(TemplateGenerator.java:1226)
	at jetbrains.mps.generator.impl.TemplateGenerator$FullCopyFacility.copyInputNode(TemplateGenerator.java:1226)
	at jetbrains.mps.generator.impl.TemplateGenerator$FullCopyFacility.copyInputNode(TemplateGenerator.java:1226)
	at jetbrains.mps.generator.impl.TemplateGenerator$FullCopyFacility.copyInputNode(TemplateGenerator.java:1226)
	at jetbrains.mps.generator.impl.TemplateGenerator.copyNodes(TemplateGenerator.java:696)
	at jetbrains.mps.generator.impl.TemplateExecutionEnvironmentImpl.copyNodes(TemplateExecutionEnvironmentImpl.java:151)
	at jetbrains.mps.generator.impl.TemplateProcessor$CopySrcMacros.apply(TemplateProcessor.java:407)
	at jetbrains.mps.generator.impl.TemplateProcessor.applyMacro(TemplateProcessor.java:112)
	at jetbrains.mps.generator.impl.TemplateProcessor.apply(TemplateProcessor.java:102)
	at jetbrains.mps.generator.impl.TemplateContainer.apply(TemplateContainer.java:77)
	at jetbrains.mps.generator.impl.RuleConsequenceProcessor$TemplateConsequence.processRuleConsequence(RuleConsequenceProcessor.java:249)
	at jetbrains.mps.generator.impl.interpreted.TemplateReductionRuleInterpreted.apply(TemplateReductionRuleInterpreted.java:74)
	at jetbrains.mps.generator.template.DefaultQueryExecutionContext.applyRule(DefaultQueryExecutionContext.java:219)
	at jetbrains.mps.generator.impl.TemplateExecutionEnvironmentImpl.tryToReduce(TemplateExecutionEnvironmentImpl.java:412)
	at jetbrains.mps.generator.impl.TemplateExecutionEnvironmentImpl.trySwitch(TemplateExecutionEnvironmentImpl.java:196)
	at jetbrains.mps.generator.impl.TemplateProcessor$SwitchMacro.apply(TemplateProcessor.java:756)
	at jetbrains.mps.generator.impl.TemplateProcessor.applyMacro(TemplateProcessor.java:112)
	at jetbrains.mps.generator.impl.TemplateProcessor.applyTemplate(TemplateProcessor.java:137)
	at jetbrains.mps.generator.impl.TemplateProcessor.applyTemplate(TemplateProcessor.java:139)
	at jetbrains.mps.generator.impl.TemplateProcessor.apply(TemplateProcessor.java:104)
	at jetbrains.mps.generator.impl.TemplateContainer.apply(TemplateContainer.java:77)
	at jetbrains.mps.generator.impl.RuleConsequenceProcessor$TemplateConsequence.processRuleConsequence(RuleConsequenceProcessor.java:249)
	at jetbrains.mps.generator.impl.interpreted.TemplateReductionRuleInterpreted.apply(TemplateReductionRuleInterpreted.java:74)
	at jetbrains.mps.generator.template.DefaultQueryExecutionContext.applyRule(DefaultQueryExecutionContext.java:219)
	at jetbrains.mps.generator.impl.TemplateExecutionEnvironmentImpl.tryToReduce(TemplateExecutionEnvironmentImpl.java:412)
	at jetbrains.mps.generator.impl.TemplateExecutionEnvironmentImpl.trySwitch(TemplateExecutionEnvironmentImpl.java:196)
	at jetbrains.mps.generator.impl.TemplateProcessor$SwitchMacro.apply(TemplateProcessor.java:756)
	at jetbrains.mps.generator.impl.TemplateProcessor.applyMacro(TemplateProcessor.java:112)
	at jetbrains.mps.generator.impl.TemplateProcessor.apply(TemplateProcessor.java:102)
	at jetbrains.mps.generator.impl.TemplateContainer.apply(TemplateContainer.java:77)
	at jetbrains.mps.generator.impl.RuleConsequenceProcessor$TemplateConsequence.processRuleConsequence(RuleConsequenceProcessor.java:249)
	at jetbrains.mps.generator.impl.interpreted.TemplateReductionRuleInterpreted.apply(TemplateReductionRuleInterpreted.java:74)
	at jetbrains.mps.generator.template.DefaultQueryExecutionContext.applyRule(DefaultQueryExecutionContext.java:219)
	at jetbrains.mps.generator.impl.TemplateExecutionEnvironmentImpl.tryToReduce(TemplateExecutionEnvironmentImpl.java:412)
	at jetbrains.mps.generator.impl.TemplateExecutionEnvironmentImpl.tryToReduce(TemplateExecutionEnvironmentImpl.java:349)
	at jetbrains.mps.generator.impl.TemplateGenerator$FullCopyFacility.copyInputNode(TemplateGenerator.java:1208)
	at jetbrains.mps.generator.impl.TemplateGenerator$FullCopyFacility.copyInputNode(TemplateGenerator.java:1226)
	at jetbrains.mps.generator.impl.TemplateGenerator$FullCopyFacility.copyInputNode(TemplateGenerator.java:1226)
	at jetbrains.mps.generator.impl.TemplateGenerator.copyNodeAttributes(TemplateGenerator.java:877)
	at jetbrains.mps.generator.impl.TemplateGenerator.createRootNodeByRule(TemplateGenerator.java:408)
	at jetbrains.mps.generator.impl.TemplateGenerator.applyRootRule(TemplateGenerator.java:385)
	at jetbrains.mps.generator.impl.TemplateGenerator.applyRootRule(TemplateGenerator.java:380)
	at jetbrains.mps.generator.impl.TemplateGenerator.applyReductions(TemplateGenerator.java:293)
	at jetbrains.mps.generator.impl.TemplateGenerator.apply(TemplateGenerator.java:181)
	at jetbrains.mps.generator.impl.GenerationSession.executeMajorStepInternal(GenerationSession.java:461)
	at jetbrains.mps.generator.impl.GenerationSession.executeMajorStep(GenerationSession.java:417)
	at jetbrains.mps.generator.impl.GenerationSession.processGenPlanBranch(GenerationSession.java:282)
	at jetbrains.mps.generator.impl.GenerationSession.generateModel(GenerationSession.java:198)
	at jetbrains.mps.generator.impl.GenerationController.generateModel(GenerationController.java:136)
	at jetbrains.mps.generator.impl.GenerationController.generate(GenerationController.java:72)
	at jetbrains.mps.generator.GenerationFacade.process0(GenerationFacade.java:178)
	at jetbrains.mps.generator.GenerationFacade.process(GenerationFacade.java:161)
	at jetbrains.mps.lang.core.plugin.Generate_Facet$Target_generate$1$4.run(Generate_Facet.java:460)
	at jetbrains.mps.smodel.ActionDispatcher.dispatch(ActionDispatcher.java:88)
	at jetbrains.mps.smodel.WorkbenchModelAccess.lambda$runReadAction$0(WorkbenchModelAccess.java:90)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:869)
	at jetbrains.mps.smodel.WorkbenchModelAccess.runReadAction(WorkbenchModelAccess.java:87)
	at jetbrains.mps.smodel.ModelAccessBase.runReadAction(ModelAccessBase.java:56)
	at jetbrains.mps.lang.core.plugin.Generate_Facet$Target_generate$1.execute(Generate_Facet.java:456)
	at jetbrains.mps.internal.make.runtime.script.Script$3.invoke(Script.java:311)
	at jetbrains.mps.internal.make.runtime.script.Script$3.invoke(Script.java:255)
	at jetbrains.mps.ide.make.WorkbenchMakeService$Controller.runJobWithMonitor(WorkbenchMakeService.java:313)
	at jetbrains.mps.internal.make.runtime.script.Script.executeTargets(Script.java:255)
	at jetbrains.mps.internal.make.runtime.script.Script.execute(Script.java:233)
	at jetbrains.mps.make.service.CoreMakeTask$1.invoke(CoreMakeTask.java:103)
	at jetbrains.mps.make.service.CoreMakeTask$1.invoke(CoreMakeTask.java:76)
	at jetbrains.mps.make.dependencies.MakeSequence.iterate(MakeSequence.java:56)
	at jetbrains.mps.make.service.CoreMakeTask.doRun(CoreMakeTask.java:76)
	at jetbrains.mps.ide.make.MakeTask$WorkbenchMakeTask.doRun(MakeTask.java:148)
	at jetbrains.mps.make.service.CoreMakeTask.run(CoreMakeTask.java:55)
	at jetbrains.mps.ide.make.MakeTask$2.run(MakeTask.java:68)
	at java.base/java.lang.Thread.run(Thread.java:834)
````

**Comment by WimBast** [comment]
A clear bug. We wil fix it. Dit you use version 1.4.1?

**Comment by enikao** [comment]
yes, version 1.4.1

**Comment by WimBast** [comment]
fixed in v1.5.0
